package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "hangoutsidesf",
+  "version": "1.0.0",
+  "description": "[Contributing Guidelines](CONTRIBUTING.md)",
+  "main": "server/index.js",
+  "scripts": {
+    "build": "webpack",
+    "server-dev": "nodemon server/index.js",
```

Comment:
script for dev should be `start:dev`

Reply:
Updated development script from `server-dev` to `start:dev`.